pkg/asset/installconfig/ssh.go

```diff
@@ -50,66 +47,28 @@ func (a *sshPublicKey) Generate(map[asset.Asset]*asset.State) (state *asset.Stat
 				continue
 			}
 
-			pubKeys[path] = pubKey
-		}
-
-		paths = []string{}
-		for path := range pubKeys {
-			paths = append(paths, path)
+			pubKeys[path] = pubKeyBytes
 		}
-		sort.Strings(paths)
 	}
 
-	promptLines := []string{"SSH Public Key:"}
-	if len(paths) == 0 {
-		promptLines = append(
-			promptLines,
-			"Enter an empty string or your public key (e.g. 'ssh-rsa AAAA...')",
-		)
-	} else {
-		promptLines = append(
-			promptLines,
-			"Enter an empty string, your public key (e.g. 'ssh-rsa AAAA...'), or one of the following numbers:",
-		)
-		for i, path := range paths {
-			promptLines = append(
-				promptLines,
-				fmt.Sprintf("%d: %s", i+1, path),
-			)
-		}
+	var paths []string
+	for path := range pubKeys {
+		paths = append(paths, path)
 	}
-	prompt := strings.Join(promptLines, "\n")
+	sort.Strings(paths)
 
-	var input string
-	for {
-		fmt.Println(prompt)
-		input, err = a.inputReader.ReadString('\n')
-		if err != nil && err != io.EOF {
-			fmt.Println("Could not understand response. Please retry.")
-			continue
-		}
-		if input != "" && input[len(input)-1] == '\n' {
-			input = input[:len(input)-1]
-		}
-		var i int
-		n, err := fmt.Sscanf(input, "%d", &i)
-		if n == len(input) && err == nil && i > 0 && i <= len(paths) {
-			path := paths[i-1]
-			input = pubKeys[path]
-		} else {
-			err = validate.OpenSSHPublicKey(input)
-			if err != nil {
-				fmt.Println(err)
-				continue
-			}
-		}
-		break
-	}
+	var path string
+	survey.AskOne(&survey.Select{
+		Message: "SSH Public Key",
+		Help:    "The SSH key used to access all nodes within the cluster. This is optional.",
+		Options: paths,
+		Default: none,
+	}, &path, nil)
 
 	return &asset.State{
-		Contents: []asset.Content{
-			{Data: []byte(input)},
-		},
+		Contents: []asset.Content{{
+			Data: pubKeys[path],
```

Comment:
Looks like we're still not supporting "paste in your SSH pubkey".  Is that going to be another "edit your install-config YAML afterwards if you need it"?

Reply:
Yeah. Copying from above because I'm lazy:

> We can just tell the user to edit their install config if they want to do that. Remember, this interactive UI is for the _basic_ use case - nothing fancy.